CODEOWNERS

```diff
@@ -1,5 +1,228 @@
 # CODEOWNERS for autoreview assigning in github
 
 # Get all docs reviewed
-doc/	@dbkinder
+doc/*	@dbkinder
 *.rst	@dbkinder
+
+* kumar.gala@linaro.org anas.nashif@intel.com
+
+arch/arc/* cjordan@synopsys.com Ruud.Derwig@synopsys.com
+include/arch/arc/* cjordan@synopsys.com Ruud.Derwig@synopsys.com
+boards/arc/* cjordan@synopsys.com Ruud.Derwig@synopsys.com
+
+arch/arm/* maureen.helm@nxp.com kumar.gala@linaro.org
+include/arch/arm/* maureen.helm@nxp.com kumar.gala@linaro.org
+boards/arm/* maureen.helm@nxp.com kumar.gala@linaro.org
+
+ext/hal/cmsis/* maureen.helm@nxp.com kumar.gala@linaro.org
+
+boards/arc/arduino_101_sss/* anas.nashif@intel.com
+
+boards/arc/em_starterkit/* cjordan@synopsys.com
+
+boards/arc/quark_se_c1000_ss_devboard/* anas.nashif@intel.com
+
+boards/arm/96b_carbon/* ricardo.salveti@linaro.org amit.kucheria@linaro.org
+
+boards/arm/96b_nitrogen/* amit.kucheria@linaro.org
+
+boards/arm/arduino_101_ble/* johan.hedberg@intel.com
+
+boards/arm/cc3200_launchxl/* gil.pitney@linaro.org
+boards/arm/cc3220sf_launchxl/* gil.pitney@linaro.org
+
+boards/arm/frdm_k64f/* maureen.helm@nxp.com
+
+boards/arm/frdm_kw41z/* maureen.helm@nxp.com
+
+boards/arm/hexiwear_k64/* maureen.helm@nxp.com
+
+boards/arm/nrf51_blenano/* ricardo.salveti@linaro.org
+
+boards/arm/nrf52_pca10040/* carles.cufi@nordicsemi.no
+
+boards/arm/nucleo_f401re/* ricardo.salveti@linaro.org amit.kucheria@linaro.org
+
+boards/arm/v2m_beetle/* vincenzo.frascino@linaro.org
+
+boards/nios2/altera_max10/* andrew.p.boie@intel.com
+
+boards/x86/arduino_101/* anas.nashif@intel.com
+
+boards/x86/galileo/* anas.nashif@intel.com
+
+boards/x86/quark_d2000_crb/* anas.nashif@intel.com
+
+boards/x86/quark_se_c1000_devboard/* anas.nashif@intel.com
+
+subsys/bluetooth/* szymon.janc@gmail.com johan.hedberg@intel.com luiz.dentz@gmail.com
+include/bluetooth/* szymon.janc@gmail.com johan.hedberg@intel.com luiz.dentz@gmail.com
+include/drivers/bluetooth/* szymon.janc@gmail.com johan.hedberg@intel.com luiz.dentz@gmail.com
+drivers/bluetooth/* szymon.janc@gmail.com johan.hedberg@intel.com luiz.dentz@gmail.com
+samples/bluetooth/* szymon.janc@gmail.com johan.hedberg@intel.com luiz.dentz@gmail.com
+tests/bluetooth/* szymon.janc@gmail.com johan.hedberg@intel.com luiz.dentz@gmail.com
+doc/subsystems/bluetooth/* szymon.janc@gmail.com johan.hedberg@intel.com luiz.dentz@gmail.com
+
+subsys/bluetooth/controller/* carles.cufi@nordicsemi.no vinayak.kariappa.chettimada@nordicsemi.no
+
+ext/hal/ti/cc3200sdk/* gil.pitney@linaro.org
+
+arch/arm/soc/ti_simplelink/cc32xx gil.pitney@linaro.org
+
+doc/* david.b.kinder@intel.com inaky.perez-gonzalez@intel.com
+*.rst david.b.kinder@intel.com inaky.perez-gonzalez@intel.com
```

Comment:
if you are hand tweaking the results, should probably remove these (as we have things above).